common/util/src/main/java/org/eclipse/dataspaceconnector/common/annotations/ExcludeFromCodeCoverageGeneratedReport.java

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2021 Copyright Holder (Catena-X Consortium)
+//
+// See the AUTHORS file(s) distributed with this work for additional
+// information regarding authorship.
+//
+// See the LICENSE file(s) distributed with this work for
+// additional information regarding license terms.
+//
+package org.eclipse.dataspaceconnector.common.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Exclude a block of code from code coverage.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface ExcludeFromCodeCoverageGeneratedReport {
```

Comment:
explain in javadoc why this works (Jacoco matches any annotation containing the "Generated" substring)

Reply:
Updated. thanks